Standardize facilities package structures

gerfboy opened the pull request:
README.md explains how to run the update script

mattnichols commented on encryption-service-jasypt/README.md:
````diff
@@ -6,27 +6,24 @@ Vault encryption service implementation
 ```yaml
 encryptionService:
-  class: "com.mx.path.facility.security.vault.VaultEncryptionService"
+  class: com.mx.path.service.facility.encryption.jasypt.JasyptEncryptionService
````
I think this should be `com.mx.path.service.facility.encryption.JasyptEncryptionService`, maybe?

mattnichols commented on upgrades/packageStructure/packageStructureUpdate.sh:
```diff
@@ -0,0 +1,108 @@
+### Driver
+export -f process_file
+export DRY_RUN
+find "$ROOT_DIR" -type f \( -iname "*.yaml" \) -exec bash -c 'process_file "$0"' {} \;
```
We might want to match `*.yml` as well

gerfboy replied:
I've added support for .yml files.

mattnichols commented on upgrades/packageStructure/README.md:
```diff
@@ -0,0 +1,9 @@
+## Package Structure Update
```
Let's rename this dir and scripts to the version, like we did in web and core.

gerfboy replied:
I wasn't sure what version this would end up being. I think it'll be 2.1.0 since I used fix!: Let me know if this isn't correct.

mattnichols replied:
It'll be 3.0.0

mattnichols replied:
Breaking changes bump the major version.

gerfboy replied:
Bumped the name of the script to 3.0.0

Commit:
fix!: standardize package structures

README.md explains how to run the update script
fixing build


Cleaning up imports


Renaming script, supporting .yml


Actual version

mattnichols commented:
Let's merge this into a v3 branch. Otherwise, looks good.
